index.bs

```diff
@@ -626,11 +626,15 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 
 			1. Let |p| be a new [=Promise=].
 
+			1. If the [=current settings object's=] responsible document is not
+				[=allowed to use=] the "clipboard-write" feature, then reject |p|
+				with a "NotAllowedError" DOMException.
+
```

Comment:
This current algorithm currently continues with the rejected promise, instead of returning it. Let's do: 

```suggestion
			1. If the [=current settings object's=] responsible document is not
				[=allowed to use=] the "clipboard-write" feature, then
				return a promise rejected with a "NotAllowedError" DOMException.

			1. Let |p| be a new [=Promise=].


```

Reply:
Just some nits... I think you can link [=responsible document=] and [=a promise rejected with=] ... please confirm by looking up the terms at https://respec.org/xref/ ... you can probably link {{DOMException}} too.